src/WLGDPSTime.cc

```diff
@@ -0,0 +1,77 @@
+// WLGDPSTime
+#include "WLGDPSTime.hh"
+#include "G4UnitsTable.hh"
+
+////////////////////////////////////////////////////////////////////////////////
+// Description:
+//   This is a primitive scorer class for scoring global track time
+//
+///////////////////////////////////////////////////////////////////////////////
+
+WLGDPSTime::WLGDPSTime(G4String name, G4int depth)
+: G4VPrimitiveScorer(name, depth)
+, HCID(-1)
+, EvtMap(0)
+{
+    SetUnit("ns");
+}
+
+WLGDPSTime::WLGDPSTime(G4String name, const G4String& unit, G4int depth)
+: G4VPrimitiveScorer(name, depth)
+, HCID(-1)
+, EvtMap(0)
+{
+    SetUnit(unit);
+}
+
+WLGDPSTime::~WLGDPSTime() { ; }
```

Comment:
```suggestion
WLGDPSTime::~WLGDPSTime() = default;
```

Or declare as `default` in the header